LICENSE

```diff
@@ -186,7 +186,7 @@ Apache License
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright {yyyy} {name of copyright owner}
+   Copyright 2015 Jeff May
```

Comment:
Um...

Reply:
I can update this before transferring ownership of the repo.